src/Services/MobitechImporter.php

```diff
@@ -27,8 +28,34 @@ public function __construct()
     public function import(string $id, string $file)
     {
         if (strpos($file, 'softpay') !== false) {
-            // Ignoring these transactions for now.
-            return 0;
+            $softpay = json_decode(file_get_contents($file));
+            SoftpayTransaction::create([
+                'chunk_date'            => new Carbon($id),
+                'operator_reference'    => strtoupper($softpay->OperatorReference),
```

Comment:
I see a repetitive pattern here.

A suggestion here is to have a methods matching the file name and call it if it exists (be aware of pseudo-code):

```php
public function import()
{
  $method = "import" . Str::studly(getSubjectInFilename($file))
  if (method_exists($this, $method) {
    $json = json_decode(...);
    call_user_func([$this, $method], $json);
  }
}

protected function importSoftpay() { 
  //
}

protected function importOrca() {
  //
}

// ...
```